tests/test_optimization/test_feedback_loop.py

```
@@ -858,3 +844,148 @@ def test_int_module_imports(self):
+class TestChromaObservabilityInStats:
+    """C3: get_learning_stats() must expose ChromaDB health so operators can
+    detect a failed init without reading logs.
+
+    Concrete scenario: ChromaDB directory is missing on startup. The sentinel
+    is set. An operator calls GET /api/learning-stats to understand why semantic
+    search is returning empty results. The "execution_memory" key tells them
+    exactly what went wrong.
+    """
+
+    def test_learning_stats_includes_execution_memory_key(self, in_memory_db):
+        fl, em, fa, se, ke, ae, mt, cd, conn = _build_feedback_loop(in_memory_db)
```

Ruff (0.15.14)

[warning] Unpacked variable `em` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)